### ISSUE TITLE
Encountered with Nan loss when training classification task

### ISSUE THREAD

**Issue by jin-yc10** [OpenGVLab/DCNv4#35]
Encountered with Nan loss when training classification task on imagenet.

Cuda: 11.6
Torch: 1.12.1+cu116
Timm: 0.6.11

Any help is appreciated! Thanks

**Comment by Ckkxx** [comment]
Yes, nan values appear when using DCNV4 instead of the normal volume downsampling operation, which is the problem I encountered

**Comment by fuyannnnn** [comment]
How did you solve this problem?

**Comment by 87831743Sakura** [comment]
我也遇到了这个问题，请问有解决方案吗？

**Comment by 87831743Sakura** [comment]
I also encountered this problem, is there a solution?

**Comment by Byte247** [comment]
Same here, Nan during training even with stride == 1
